afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/UuidList.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.mapdb.storage;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class UuidList {
+
+    private final List<UUID> uuids;
+
+    public UuidList() {
+        this(new ArrayList<>());
+    }
+
+    public UuidList(List<UUID> uuids) {
+        this.uuids = Objects.requireNonNull(uuids);
+    }
+
+    public int size() {
+        return uuids.size();
+    }
+
+    public UUID get(int i) {
+        return uuids.get(i);
+    }
+
+    public List<UUID> toList() {
+        return uuids;
```

Comment:
Is it safe to return this list instead of an unmodifiable copy?
If this method is only use to get a stream over the uuids list, maybe you can change it to a stream() method

Reply:
This is an internal class, so it is safe.